packages/expo-contacts/android/src/main/java/expo/modules/contacts/ContactsModule.kt

```diff
@@ -644,6 +810,39 @@ class ContactsModule : Module() {
     return input
   }
 
+  private fun fetchGroups(
+    queryStrings: Array<String>?,
+    initQueryField: String?
+  ): List<Group>? {
+    val queryField = initQueryField ?: ContactsContract.Groups._ID
+    val groups: Map<String, Group>
+    val cr = resolver
+
+    if (!queryStrings.isNullOrEmpty()) {
+      val cursorSelection = "$queryField LIKE ?"
+      cr.query(
+        ContactsContract.Groups.CONTENT_URI,
+        arrayOf(ContactsContract.Groups._ID, ContactsContract.Groups.TITLE),
+        cursorSelection,
+        queryStrings,
+        null
+      )
+    } else {
+      cr.query(
+        ContactsContract.Groups.CONTENT_URI,
+        arrayOf(ContactsContract.Groups._ID, ContactsContract.Groups.TITLE),
+        null,
+        null,
+        null
+      )
+    }?.use { cursor ->
+      groups = loadGroupsFrom(cursor)
+
```

Comment:
```suggestion
```

Reply:
Thank you for your review on this. I have updated the typescript documentation now.
This is the only change I am unsure about exactly what you are asking me to do, thanks.